python/pyspark/errors/error_classes.py

```diff
@@ -536,6 +551,21 @@
       "Result vector from pandas_udf was not the required length: expected <expected>, got <actual>."
     ]
   },
+  "SESSION_OR_CONTEXT_EXISTS" : {
+    "message" : [
+      "There should not be an existing Spark Session or Spark Context."
+    ]
+  },
+  "SHOULD_NOT_DATAFRAME": {
+    "message": [
+      "Argument `<arg_name>` should not be a DataFrame."
+    ]
+  },
+  "SHOULD_NOT_DATATYPE": {
+    "message": [
+      "Argument `<arg_name>` should not be a DataType."
+    ]
+  },
```

Comment:
not sure whether `SHOULD_NOT_DATAFRAME` and `SHOULD_NOT_DATATYPE ` can be consolidated to a `INVALID_TYPE`

Reply:
Sounds good! Just applied the comment :-)